### DISCUSSION TITLE
Why does neovim crash when I open a neorg file?

### DISCUSSION THREAD

**Discussion by Luxvao** [nvim-neorg/neorg#1102] [Q&A]
I installed neorg (using lazy) with this:
```
{
        "nvim-neorg/neorg",
        build = ":Neorg sync-parsers",
        dependencies = { "nvim-lua/plenary.nvim" },
        config = function()
          require("neorg").setup {
            load = {
              ["core.defaults"] = {}, -- Loads default behaviour
              ["core.concealer"] = {}, -- Adds pretty icons to your documents
              ["core.dirman"] = { -- Manages Neorg workspaces
                config = {
                  workspaces = {
                    notes = "C:\\Users\\Bor\\notes",
                  },
                  default_workspace = "notes"
                },
              },
            },
          }
        end,
},
```
But if I try to open a neorg file (nvim <file_name>) or do :Neorg index, then neovim just crashes. I'm not sure what info to provide so tell me if you need more info.

**Comment by max397574** [comment]
no matter what plugins do, neovim should never crash
so likely this is a neovim issue
read the neovim wiki to see how to get a stacktrace and open an issue there
you can also send it here so we can see if we can avoid it

**Comment by verajosemanuel** [comment]
Same issue here. Not a Neovim issue. With neorg plugin inactive nvim worked flawlessly. Activate reorg and having a crash is the same. Tested on a fresh new installation of LazyVim with no other plugins added. iTerm on macOS Sonoma here.

**Reply by max397574** [reply]
well as said
neovim should never crash no matter what plugins do
and also as said in the above comment
get a stacktrace as shown in the neovim wiki so we can see where this originates from

otherwise there is no way we can do anything about this because apparently it works for the vast majority of users

**Comment by massix** [comment]
Which version of `Neovim` are you using? I had some issues with the nightly version (`0.10.something`) and Treesitter, with constant crashes when loading some grammars (especially `just`). Reverted back to nvim 0.9.5 and everything was fine.
